$NetBSD: patch-machine_sun4_SUN4-75,v 1.2 2020/03/04 07:52:11 rin Exp $

Add tap(4) support for *BSD by kiyohara@:
https://mail-index.netbsd.org/port-sparc64/2013/01/29/msg001951.html

Make it use auto-allocated ptys

--- machine/sun4/SUN4-75.orig	2020-03-04 16:33:49.000533746 +0900
+++ machine/sun4/SUN4-75	2020-03-04 16:34:00.445646554 +0900
@@ -46,14 +46,14 @@
 
 ## ttya:
 ##
-# comment the first line, and uncomment the following line if you
-# aren't using the GTK-based display console.  this will connect ttya
-# to the master side of a pseudo-tty (in this case, /dev/ttyr0), which
-# you can connect to using tip or some other serial communications
-# program on the slave side (in this case, /dev/ptyr0):
+# uncomment the following line if you aren't using the GTK-based
+# display console above.  this will connect ttya to the master side of
+# a pseudo-tty, which you can connect to using tip or some other serial
+# communications program on the slave (in this case, a pty allocated
+# automatically):
 #
 sink1 at zs0 channel A: tme/serial/sink
-#console0 at zs0 channel A: tme/host/posix/serial device /dev/ttyr0 break-carats
+#console0 at zs0 channel A: tme/host/posix/serial device pty
 
 ## the SBus esp SCSI:
 ##
@@ -69,6 +69,7 @@
 #
 le0 at sbus0 slot 0 offset 0xc00000 ipl 5 dma-offset 0xff000000: tme/ic/am7990
 #bpf0 at le0: tme/host/bsd/bpf
+#tap0 at le0: tme/host/bsd/tap
 
 ## an SBus cgthree:
 ##
